argopy/xarray.py

```diff
@@ -765,38 +765,41 @@ def mid(x):
         that = []
         if 'SA' in vlist:
             SA = xr.DataArray(sa, coords=this['PSAL'].coords, name='SA')
-            SA.attrs['standard_name'] = 'Absolute Salinity'
+            SA.attrs['long_name'] = 'Absolute Salinity'
+            SA.attrs['standard_name'] = 'sea_water_absolute_salinity'
             SA.attrs['unit'] = 'g/kg'
             that.append(SA)
 
         if 'CT' in vlist:
             CT = xr.DataArray(ct, coords=this['TEMP'].coords, name='CT')
-            CT.attrs['standard_name'] = 'Conservative Temperature'
+            CT.attrs['long_name'] = 'Conservative Temperature'
```

Comment:
Thanks for preserving these as long names, it will reduce breakages in case people are relying on those names.

Reply:
As much as the various data standards "recommend against it" I think most people rely on the variable name itself (in this case it's `CT`) rather than things in attributes. I'd like to get some doc string things added before this is merged mainly defining all the options for `vlist`.

I'll also note the potentially breaking change in the docs themselves.